deploy/k8s/charts/trustification/report-config/.htpasswd

```diff
@@ -0,0 +1 @@
+admin:$apr1$XFCFfzu4$vtfox1qeBPqcZv814kltL1
```

Comment:
The next step is to make this configurable for the person deploying the application.